extensions/guacamole-auth-cas/src/main/resources/guac-manifest.json

```diff
@@ -13,7 +13,8 @@
         "translations/de.json",
         "translations/en.json",
         "translations/ja.json",
-        "translations/ru.json"
+        "translations/ru.json",
+        "translations/ca.json"
```

Comment:
This should be added in alphabetical order, like the rest of the list. So, should go at the start of the list.

Reply:
Sorry, I reorder the content of the translation list in alphabetical order.

Thank you very much for the clarification.